src/trackers/SHRI.py

```diff
@@ -30,6 +43,68 @@ async def get_name(self, meta):
         source = meta.get('source', "")
         imdb_info = meta.get('imdb_info') or {}
 
+        if not meta.get('is_disc') and ('untouched' in basename.lower() or 'vu' in basename.lower()):
+            meta['type'] = 'REMUX'
```

Comment:
I'll attempt to clean this up, but for future reference, don't define the meta object like this, since it sets the meta object that everything else is relying on.

Reply:
You're setting the meta object, so when the user moves from uploading to SHRI, to the next tracker in their list, the meta object has been forcibly changed, and will be incorrect for any trackers processed afterwards.